src/clarity/iconography/demo/icon-single-select.html

```diff
@@ -0,0 +1 @@
+<clr-icon [attr.shape]="shape"></clr-icon><span class="icon-shape-name">{{shape}}</span>
```

Comment:
This component was created to remove the effort manually copy pasting each icon names next to the icons. Also, this template might grow big as we want to show icon usage of each icon in a modal.